pages/04.applications/99.wishlist/apps_wishlist.md

```diff
@@ -164,6 +164,7 @@ You can [contribute to this list by adding something you'd like to be packaged](
 | [Lessy](https://lessy.io) |  | [Upstream](https://github.com/lessy-community/lessy) |  |
 | [Librarian](https://lbry.bcow.xyz/) | An alternative frontend for LBRY/Odysee | [Upstream](https://codeberg.org/librarian/librarian) |  |
 | [LibreTranslate](https://libretranslate.com/) | Translation API | [Upstream](https://github.com/uav4geo/LibreTranslate) | [Package Draft](https://github.com/YunoHost-Apps/libretranslate_ynh) |
+| [Lichen](https://git.sensorstation.co/lichen.git) |  | [Upstream](https://git.sensorstation.co/lichen.git) |  |
```

Comment:
It would be nice if you could explain in a few (10~15 max) words what this app is, it's totally not obvious from the git repo ...

Reply:
In will fix that. Thanks!